node/src/chain_spec.rs

```diff
@@ -101,7 +101,7 @@ pub fn local_testnet_config() -> Result<ChainSpec, String> {
 		Some("Golden Gate"),
```

Comment:
@akorchyn As we rename the brand from Golden Gate to GGX, does it make sense to rename it in spec?

Reply:
Sure